### PR TITLE
grpc fixes

### PR DESCRIPTION
**Pull request by dimaxgl**
global fixes for NewGRPCOptionsFromConfig
added grpc config tests

### CODE REVIEW

File: util/grpc.go [hunk 1]
```diff
@@ -3,24 +3,25 @@ package util
 import (
 	"context"
 	"crypto/tls"
+	"crypto/x509"
 	"fmt"
-	"net"
+	"google.golang.org/grpc/balancer/roundrobin"
```
Comment (vitiko) [inline util/grpc.go:8]: imports order mixed

File: util/grpc.go [hunk 2]
```diff
@@ -48,34 +49,54 @@ func NewGRPCOptionsFromConfig(c config.ConnectionConfig, log *zap.Logger) ([]grp
...
+					return nil, errors.Wrap(err, `failed to load client certificate`)
 				}
+				tlsCfg.Certificates = append(tlsCfg.Certificates, cert)
 			} else {
-				l.Debug(`Using TLS host`, zap.String(`host`, c.Host))
-				if addr, _, err = net.SplitHostPort(c.Host); err != nil {
-					return nil, errors.Wrap(err, `failed to parse tls host`)
-				}
+				// otherwise just add cert to pool
+				//cert, err := ioutil.ReadFile(c.Tls.CertPath)
```
Comment (vitiko) [inline util/grpc.go:88]: remove commented code?

Comment (dimaxgl) [reply util/grpc.go]: @vitiko removed

File: util/grpc.go (hunk 1 above)
Comment (dimaxgl) [reply util/grpc.go]: fixed

**Review by vitiko — APPROVED** [review]

**Pull request merged**